xdem/terrain.py

```diff
@@ -109,24 +286,35 @@ def _divider_method_coef(res: float, coef: str) -> float:
     """Divider for a given coefficient based on resolution."""
 
     mapping_div_coef = {
-        "zt_a": res**4,
-        "zt_b": res**3,
-        "zt_c": res**3,
-        "zt_d": res**2,
-        "zt_e": res**2,
-        "zt_f": res**2,
-        "zt_g": res,
-        "zt_h": res,
+        "zt_a": 4 * res**4,
+        "zt_b": 4 * res**3,
+        "zt_c": 4 * res**3,
+        "zt_d": res**2,  # Divided by 2 compared to Zevenberg to match z_xx definition
+        "zt_e": res**2,  # Divided by 2 compared to Zevenberg to match z_xx definition
+        "zt_f": 4 * res**2,  # Times 2 what is reported in ZevenbergThorne because later formula multiplies by 2
+        "zt_g": 2 * res,
+        "zt_h": 2 * res,
```

Comment:
@trchudley I've removed the fractions in the ZT kernels for clearer reading, and divided by 2 where necessary to make ZT/Minar formulas consistent.